lib/ecmascript.ts

```diff
@@ -1259,19 +1435,24 @@ export function ToTemporalTime(item, overflow = 'constrain') {
   return new TemporalPlainTime(hour, minute, second, millisecond, microsecond, nanosecond);
 }
 
-export function ToTemporalYearMonth(item, options = ObjectCreate(null)) {
+export function ToTemporalYearMonth(
+  item: Temporal.PlainYearMonth | Temporal.PlainYearMonthLike | string,
+  options = ObjectCreate(null)
```

Comment:
Same question here r.e. the type of options.

Reply:
Same answer. :-)